src/components/theme-icon/NotificationIndicator.tsx

```diff
@@ -66,8 +66,8 @@ const getIndicatorSize = (
 const useStyles = StyleSheet.createThemeHook((theme) => ({
   indicator: {
     position: 'absolute',
-    right: 0,
-    top: 0,
+    right: -4,
```

Comment:
Is this half the width (and height below here) of the notification indicator?

Reply:
This was decided mainly on feel, and what looked best 😅 The notification indicator already lives a bit of a life of it's own outside of the design system.